pkgs/development/python-modules/pgpy/default.nix

```diff
@@ -7,6 +7,7 @@
   pyasn1,
   cryptography,
   pytestCheckHook,
+  standard-imghdr,
```

Comment:
I first started looking at other usages "python-deadlib" in nixpkgs, given it has a lot of modules,  but so far this is first such usage.

after reading https://peps.python.org/pep-0594/#deprecated-modules

it looks like imghdr is old and should not be used

so options ( in no particular order )

- port pgpy to use [filetype](https://pypi.org/project/filetype/), [puremagic](https://pypi.org/project/puremagic/), [python-magic](https://pypi.org/project/python-magic/)
- mark pgpy as broken on python 3.13
- add imghdr ( as in this PR)

I am not maintainer of pgpy, nor direct or indirect consumer og pgpy so I don't have any say in what should be done here